pallets/subtensor/src/lib.rs

```diff
@@ -1196,6 +1218,32 @@ pub mod pallet {
             Self::do_set_weights(origin, netuid, dests, weights, version_key)
         }
 
+        #[pallet::call_index(96)]
+        #[pallet::weight((Weight::from_parts(10_151_000_000, 0)
```

Comment:
Can we benchmark these to arrive at the weights ? Intuition says the commits will be cheaper than the previous set_weights function , and reveals more expensive